web/packages/core/src/public-api.ts

```diff
@@ -83,7 +83,7 @@ export class PublicAPI {
         }
 
         if (document.readyState === "loading") {
-            window.addEventListener("DOMContentLoaded", this.init.bind(this));
+            document.addEventListener("readystatechange", this.init.bind(this));
```

Comment:
Maybe worth meintioning in a comment that `DOMContentLoaded` isn't used because of Rocket Loader (as you did in earlier commits)?